### DISCUSSION TITLE
Replacing the manager node in a distributed environment

### DISCUSSION THREAD

**Discussion by 2manyvcos** [Security-Onion-Solutions/securityonion#8342] [Unsupported Versions]
Hello there!

We have a distributed SO environment with multiple forwarder nodes spread across the company and one managersearch node.
Due to changed requirements, we now need to replace the managersearch node by another more powerful server.

What is the recommended way to achieve this?
Can I simply copy the disks to the new server using `dd`? If so, what must be done afterwards to make this work (e.g. configuring changed network interface name & MAC address)?

**Comment by McLoggin** [comment]
Second this, I would be very curious to see a process in place to migrate from managersearch node -> manager node + search node. Our managersearch is starting to hit high resource utilization, and the only (terrible) option I can think of would be to stand up a completely separate manager + search node. Then, migrate all of the forwarder nodes over to new environment, so that we don't miss out on a whole lot of data.

**Comment by dougburks** [comment]
Could you just join new search nodes to your existing manager?

**Reply by 2manyvcos** [reply]
Sadly not.

What I did now was that I configured the new server as a managersearch node in a new distributed environment and then joined the existing forwarder nodes to the new manager via SSH using the `sudo SecurityOnion/setup/so-setup iso` command.

However, with this solution, I had to sacrifice all data that had been collected.
I also had to reconfigure the Wazuh + Osquery agents on all clients.
